gulpfile.js

```diff
@@ -26,13 +42,42 @@ var basePaths = {
         bower: {
             src: './bower_components/'
         }
+    },
+    // Define names of third-party dependencies
+    libs = {
+        vendor: {
+            /* -------------------------------------------------------------------------------------
+             * core js libs - present on every site page, these will be concatenated in order into
+             * vendor.js
+             * ---------------------------------------------------------------------------------- */
+            core: {
+                jquery: {
+                    name: 'jquery'
+                }
+            },
+            /* -------------------------------------------------------------------------------------
+             * ie8- js libs - present on every site page, thse will be concatenated in order into
+             * ie8.js
+             * ---------------------------------------------------------------------------------- */
+            ie8: {
+                html5shiv: {
+                    name: 'html5shiv'
+                },
+                respond: {
+                    name: 'respond'
+                }
+            }
+        }
```

Comment:
This can probably be tidied up in future to something like
```
libs: {
    vendor: {
        core: ['jquery'],
        ie8: ['html5shiv', 'respond']
    }
}
```